### ISSUE TITLE
Ontario project data

### ISSUE THREAD

**Issue by jeffreyhanson** [NCC-CNC/wheretowork#182]
@ricschuster, just posting this issue so we don't forget, you mentioned that there might be some data sensitivity issues with the Ontario project data. Could you please follow up with the relevant people to check if it's ok that we include this dataset?

**Comment by jeffreyhanson** [comment]
@ricschuster, I just wanted to follow up on this? Veronique mentioned that she expected a number of people to start using the WTW and WTD apps following her presentation (on 24th) - so if the Ontario project contains sensitive data then this should not be accessible under the public login? That said, I think it would be a shame for the Ontario project to be removed - I think it's a pretty good project datast for introducing the tool (perhaps, if needed, we could create a public version of it without any of the confidential layers).

**Comment by ricschuster** [comment]
Thanks for bringing this up again. I'm going to ask the Onatrio staff about this to make sure we can use the Ontario data in WTW. I will ask them both on the inclusion in the public login instance as well as it being part of the WTW package.

**Comment by jeffreyhanson** [comment]
Awesome - thanks!

**Comment by jeffreyhanson** [comment]
@ricschuster, just wanted to check if there was any updates on this?

**Comment by ricschuster** [comment]
Still waiting for a decision from folks in Ontario (they are working on it).

**Comment by jeffreyhanson** [comment]
Ah ok - thanks!

**Comment by jeffreyhanson** [comment]
@ricschuster, any updates?

**Comment by ricschuster** [comment]
I've asked again. So far no yes/no.

**Comment by jeffreyhanson** [comment]
Hmm, should we remove the Ontario case study then? This would be a shame because it's the best case study we have (e.g., realistic data and solves fairly quickly).

**Comment by ricschuster** [comment]
Lets wait. I did ask them whether we should remove the case study. Until I hear otherwise, I assume we can use it.

**Comment by DanWismer** [comment]
I am in the process of creating a new ontario pilot built-in dataset that will be nested inside the `wheretowork` repo. All data used will be publicly available so we wont run into data sharing problems. Once completed, I will remove the original case study. 

![image](https://user-images.githubusercontent.com/29556279/177860756-8c80b642-4bb7-4330-bec0-43cbe83bb56b.png)

**Comment by DanWismer** [comment]
I have removed the original ontario pilot data / files from the `inst/extdata` folder and have added the new `south-western-ontario` example that will ship with the tool.

@jeffreyhanson @ricschuster 
I now need to move the project files over to the server. I am wondering, since the data is shipped with the app itself; it  should find the built-in project once I merge PR #220  (changes on production will be seen the next day)?

In other words, did you have to copy over the original `inst/extdata/ontario-pilot-data` to the server? Or rather, since it lives inside the app, that is taken care of already?

**Comment by DanWismer** [comment]
And yes, I had to copy the project to the server. I forgot that it was only web-apps that we cloned to the server (nested in our user account)
